### PR TITLE
[Merged by Bors] - Refactor test_bs test file

### LINKED ISSUE #1867: Refactor test_bs
# Overview / Motivation
refactor test_bs to contain only mining tests.
test_bs contains setup functions and all fixtures, both should be removed from the file.

### PR DESCRIPTION
**Pull request by AmitShaul**
## Motivation
`test_bs.py` is a mess, all setup functions and basic fixtures are in it.

## Changes
Refactor this file to contain only mining tests.

### CODE REVIEW

**Comment by AmitShaul** [comment]
bors try

**Comment by AmitShaul** [comment]
bors try

**Comment by AmitShaul** [comment]
bors try

**Comment by AmitShaul** [comment]
bors try

**Comment by AmitShaul** [comment]
bors try

**Comment by AmitShaul** [comment]
bors try

File: tests/conftest.py [hunk 1]
```diff
@@ -131,3 +138,140 @@ def init_session(load_config, set_namespace, set_docker_images, session_id):
...
+    if not match:
+        raise Exception("Failed to read container logs in {0}".format("poet"))
+
+    print("Starting PoET")
+    out = api_call(bs_pod['pod_ip'], '{ "gatewayAddresses": ["127.0.0.1:9091"] }', 'v1/start', namespace, "80")
+    assert out == "{}", "PoET start returned error {0}".format(out)
+    print("PoET started")
+
+
+# The following fixture is currently not in used and mark for deprecation
```
Comment (noamnelke) [inline tests/conftest.py]: Please remove this commend and preferably also the unused, deprecated method.

**Review by noamnelke — CHANGES_REQUESTED** [review]
LGTM, other than that one comment.

**Review by noamnelke — APPROVED** [review]

File: tests/conftest.py (hunk 1 above)
Comment (AmitShaul) [reply tests/conftest.py]: removed in aada8c1fbe2643506f7d34c0fa022093e17d9d2a

**Comment by AmitShaul** [comment]
bors merge

**Pull request closed without merging**